src/app/js/public/facet/AppliedFacetList.js

```diff
@@ -26,18 +30,29 @@ export const AppliedFacetListComponent = ({ facets }) => (
                         value={value}
                     />
                 ))}
+                {facets.length && (
+                    <Chip style={styles.chip} onClick={clearAll}>
+                        Clear All
+                    </Chip>
+                )}
             </div>
         ) : null}
-        <Stats />
     </div>
 );
 
 AppliedFacetListComponent.propTypes = {
     facets: PropTypes.arrayOf(PropTypes.any).isRequired,
+    clearAll: PropTypes.func.isRequired,
 };
 
 const mapStateToProps = state => ({
     facets: fromFacet.getAppliedFacetList(state),
 });
 
-export default connect(mapStateToProps)(AppliedFacetListComponent);
+const mapDispatchToProps = {
+    clearAll: () => clearFacet(),
```

Comment:
why use a different name?

Reply:
to emphasize that this call will clear all facet and not just one

Reply:
And also to not pass click event in the call.